src/runtime/mod.rs

```diff
@@ -533,32 +522,25 @@ impl Module {
     // contains an HTTP error. This can occur, for example, if the WASM module sets
     // the status code on its own.
     //
-    // TODO: Waaaay too many args
+
     #[allow(clippy::too_many_arguments)]
-    #[instrument(level = "info", skip(self, req, body, client_addr, cache_config_path, cache_dir, base_log_dir, default_host), fields(uri = %req.uri, module = %self.module))]
+    #[instrument(level = "info", skip(self, req, body, info), fields(uri = %req.uri, module = %self.module))]
```

Comment:
Can you add back in the `use_tls` and `env` fields to the log entry? You can add them to `fields(...)`

Reply:
done

Reply:
Doesnt let me pass info.env_vars as a param since it is a HashMap

Reply:
You can pass it with the debug sigil before it: `?info.env_vars`

Reply:
done